code/mob/living.dm

```diff
@@ -659,6 +661,7 @@
 				src.emote("handpuppet")
 
 /mob/living/say(var/message, ignore_stamina_winded)
+	singing=0;
```

Comment:
why are you resetting the value to 0 here every time?

Reply:
Perhaps it's clearer if I move it here:
![image](https://user-images.githubusercontent.com/7035767/96355502-07164580-112e-11eb-9239-279a77571dce.png)